src/component/input.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import ValidationSchema from '../config/ValidationSchema';
+
+class Input extends React.Component {
+
+  state = {
+    value: '',
+    errorMesssage: ''
+  };
+
+  InputValidation = () => {
+
+    const { inputType, handlechange, name } = this.props;
+    const { value } = this.state;
+    const { regex, errorMsg } = ValidationSchema[inputType];
+    if (!value) {
+      this.setState({
+        errorMesssage: 'Field can not be empty'
+      });
+    } else if (!regex.test(value)) {
+      this.setState({
+        errorMesssage: errorMsg
+      });
```

Comment:
@RajanTank put setState out of the if statement and store message into variable and then set it into state